lineman/app/components/thread_page/activity_card/activity_card.coffee

```diff
@@ -75,15 +75,7 @@ angular.module('loomioApp').directive 'activityCard', ->
       item.sequenceId < $scope.discussion.lastSequenceId
 
     $scope.safeEvent = (kind) ->
-      _.contains ['new_comment',
-                  'new_motion',
-                  'new_vote',
-                  'motion_closed',
-                  'motion_closed_by_user',
-                  'motion_edited',
-                  'motion_outcome_created',
-                  'motion_outcome_updated',
-                  'discussion_edited'], kind
+      _.contains AppConfig.safeThreadItemKinds, kind
```

Comment:
This way we can be sure that the events that come back in `/api/v1/events?discussion_id=:id` are the same as the ones being displayed in the activity card, because the definition of `discussion.items` is 'an event which has discussion id `<discussion.id>` and kind within `<Discussion::THREAD_ITEM_KINDS>`

(We also might be able to just lose this, but I'd rather keep it there for now)